extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaJtaConfiguration.java

```diff
@@ -9,7 +10,7 @@
 /**
  *
  */
-@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+@ConfigRoot(name = "transaction-manager", phase = ConfigPhase.RUN_TIME)
 public final class NarayanaJtaConfiguration {
```

Comment:
Let's rename the class to `TransactionManagerConfiguration` instead of specifying a name?

Reply:
OK, done